apps/tenant-management-webapp-e2e/src/integration/events.feature

```diff
@@ -102,4 +102,31 @@ Feature: Events
     And the user selects "Streams" tab for "Event"
     Then the user views Core streams section
     When the user clicks eye icon of "PDF generation updates" under Core streams
-    Then the user views the details of "PDF generation updates" under Core streams
\ No newline at end of file
+    Then the user views the details of "PDF generation updates" under Core streams
+
+  @TEST_CS-1480 @REQ_CS-1352 @regression
+  Scenario: As a tenant admin, I can add, edit and delete an event stream
+    Given a tenant admin user is on event streams page
+    # Add a stream
+    When the user clicks Add stream button
+    Then the user views Add stream modal
+    When the user enters "autotest-addEditDelete-stream", "autotest-stream-desc", "status-service:application-healthy, file-service:file-deleted", "public" in stream modal
```

Comment:
When the user enters "autotest-addEditDelete-stream", "autotest-stream-desc", "status-service:application-healthy, file-service:file-deleted", "public" in Add stream modal

Reply:
added Add to the name and also added the case where user can enter n/a